test/integration/integration.test.js

```diff
@@ -0,0 +1,216 @@
+        console.log(results)
+        done()
+      })
+    })
+  })
+  it('Should request a range of items record', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.auth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      wrapper.requestRangeItem('10000000', '', (errorBibReq, results) => {
+        if (errorBibReq) console.log(errorBibReq)
+        results.data.total.should.equal(50)
+        console.log(results)
+        console.log(JSON.stringify(results, null, 2))
+        done()
+      })
+    })
+  })
+  it('Should request multiple bib records', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+        console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+      }
+    wrapper.auth((errorAuth, results) => {
+        if (errorAuth) console.log(errorAuth)
+      wrapper.requestMultiBibBasic(['14628261','14628262','14628263','14628264','14628265','14628266','14628267','14628268','14628269','14628270'], (errorBibReq, results) => {
+          if (errorBibReq) console.log(errorBibReq)
+        results.data.total.should.equal(10)
+          // console.log(JSON.stringify(results,null,2))
+          done()
+        })
+      })
+    })
+    it('Should request multiple item records', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.auth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      wrapper.requestMultiItemBasic(['10000000','10000100','10000200','10000300','10000400','10000500','10000600','10000700','10000800','10000900','10001000'], (errorItemReq, results) => {
+        if (errorItemReq) console.log(errorItemReq)
+        results.data.total.should.equal(11)
+        // console.log(JSON.stringify(results,null,2))
+        done()
+      })
+    })
+  })
+  it('Should make a get request and then call the callback', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.promiseAuth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      return wrapper.apiGet('patrons/1001006', (errorItemReq, results) => {
+        if (errorItemReq) console.log(errorItemReq)
+        // return new Promise((resolve, reject) => {
+        //   resolve(results) })
+        return results
+      })
+    })
+    .then((results) => {
+      results.should.be.type('object')
```

Comment:
Maybe you can test that the content from `results` is what you expect besides just checking the type.

Reply:
Fixed